gravitee-apim-console-webui/src/management/api/entrypoints-v4/api-entrypoints-v4-general.component.html

```diff
@@ -61,32 +67,23 @@ <h3>Entrypoints</h3>
           <th mat-header-cell *matHeaderCellDef></th>
           <td mat-cell *matCellDef="let element" class="entrypoints__type__table__actions">
             <button type="button" mat-icon-button aria-label="Edit" (click)="onEdit(element)">
-              <mat-icon svgIcon="gio:edit-pencil"> </mat-icon>
+              <mat-icon svgIcon="gio:edit-pencil" matTooltip="Edit"> </mat-icon>
             </button>
-            <button type="button" mat-icon-button aria-label="Edit" (click)="onDelete(element)">
-              <mat-icon svgIcon="gio:trash"></mat-icon>
+            <button type="button" [disabled]="dataSource.length <= 1" mat-icon-button aria-label="Delete" (click)="onDelete(element)">
```

Comment:
I would have hide it if the action is not possible instead of changing the tooltip message. That's what has been decided with the Ui/Ux team in the endpoints page 🤷